doc/user-getting-started-tee-aware.md

```diff
@@ -224,6 +224,9 @@ First we declare the desire in the config.json:
 
     // The heap size of the user application. Increase this setting if your app experienced OOM.
     "MemorySize": "1g",
+    // The stack size for the user application's main process. If increase heap size won't solve
+    // OOM, try increase this setting, e.g. to "8m".
+    "MainStackSize": "1536k",
```

Comment:
main_stack_szie is an advanced configuration. Most developers probably do not need to bother with this configuration. The sample should use most common configuration and leave main_stack_szie as implicit default

Reply:
Fixed.